src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmBatchProcessorImpl.java

```diff
@@ -57,22 +51,19 @@ public class UnstructuredToArmBatchProcessorImpl implements UnstructuredToArmBat
 
     private static final int BLOB_ALREADY_EXISTS_STATUS_CODE = 409;
 
-    private static final String DARTS_STRING = "darts";
-    private static final String DETS_STRING = "dets";
 
     @Override
     @SuppressWarnings({"PMD.AvoidInstantiatingObjectsInLoops", "PMD.CognitiveComplexity", "PMD.CyclomaticComplexity"})
-
     public void processUnstructuredToArm(int taskBatchSize) {
 
         log.info("Started running ARM Batch Push processing at: {}", OffsetDateTime.now());
```

Comment:
any reason not to use the `currentTimeHelper` that we have available in a field already?